index.html

```diff
@@ -0,0 +1,46 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+   <body id="main">
+        <h2 id="title"> Albert Einstein</h1>
+            <p>Legend Physics Scientist</p>
+            <div id="img-div">
```

Comment:
**Required Changes ♻️**

Check the comments under the review.

- [ ]  Why you haven't used **Semantic** HTML like "section" instead of "div" I think it would be expressive more.

Reply:
- [ ] [REQUIRED] You have passed a good review regarding the use of Semantic HTML, however, kindly note to always include an explanation, a "why", for each requested change. Why should semantic HTML be used?
- [ ] Please note that your comments should be phrased politely as suggestions. For example, This is a polite suggestion: "I think it would be better to use semantic tags", "Kindly ensure to use semantic tags"

Reply:
**Required Changes ♻️**

Check the comments under the review.

- [ ]  I think it would be better to use semantic tags as it's very expressive tags that describe themselves without adding comments to explain them. For example "section" tag describes the tag as it's a section.